### PR TITLE
fleetctl 0.10.0

### PR DESCRIPTION
**Pull request by robotblake**

### CODE REVIEW

File: Library/Formula/fleetctl.rb [hunk 1]
```diff
@@ -15,6 +15,16 @@ class Fleetctl < Formula
...
 
+  def caveats; <<-EOS.undent
+    fleetctl versions >= 0.10.0 use the fleet API instead of etcd by default.
+
+    If you want to revert to previous behavior, you can use the --driver flag:
+      fleetctl --driver=etcd <cmd>
+    Or you can set it with FLEETCTL_DRIVER:
+      export FLEETCTL_DRIVER="etcd"
+    EOS
+  end
```
Comment (mikemcquaid) [inline Library/Formula/fleetctl.rb]: Can we try and slim down the number of lines here? I also wonder if we even need these given they aren't Homebrew-specific.

Comment (robotblake) [inline Library/Formula/fleetctl.rb]: Agreed, wasn't thinking about the "Homebrew-specific" angle.

**Comment by robotblake** [comment]
Caveats removed and rebased onto master.

**Comment by asparagui** [comment]
Thanks for the patch and tweaks!

**Pull request closed without merging**